crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -69,7 +69,7 @@ const PG_COMMIT_PARALLEL_CHUNK_SIZE: usize = 100;
 // The amount of rows to update in one DB transcation, for objects particularly
 // Having this number too high may cause many db deadlocks because of
 // optimistic locking.
-const PG_COMMIT_OBJECTS_PARALLEL_CHUNK_SIZE: usize = 1000;
```

Comment:
after experiment, it's faster to backfill via 500 batch instead of 1000 ones
```
Persisted 500 chunked objects history elapsed=1.131822048
```

and 

```
Persisted 1000 chunked objects history elapsed=3.182003507
```